bridge-token/src/lib.rs

```diff
@@ -83,6 +83,8 @@ impl BridgeToken {
         decimals.map(|decimals| self.decimals = decimals);
         #[cfg(feature = "migrate_icon")]
         icon.map(|icon| self.icon = Some(icon));
+        #[cfg(not(feature = "migrate_icon"))]
+        env::log("Icon is not supported".as_bytes())
```

Comment:
We should `log` this only if the icon is `Some` with something like: "Icon was provided, but it's not supported for the token"